reverse_image_search_module/search_image.py

```diff
@@ -5,11 +5,13 @@
 import pandas as pd
 import torchvision.transforms as transforms
 from PIL import Image
+from collections import Counter
+
 
 # Load all embeddings from the .npy filessssssssssssssssssss
-all_embeddings = np.load('./data/embeddings.npy')
+all_embeddings = np.load('./data/embeddings_multi.npy')
```

Comment:
I think we have a little bit too many hardcoded paths in the project now. Let’s set them up in `.env.example` and `.env`, WDYT?

Reply:
And I’d implement some project-wide configuration switch: you either use multicropping or single cropping. If it make sense to use it in several files, then it would make things simpler to change - if you change it in one place, all other places are work in sync, so you won’t encounter problems like if you forgot to turn it on in all places. Same logic as with these paths I mentioned in the previous comment. WDYT?

Reply:
Resolved in commit 05a576b :smile: